spec/models/accounting/qb/purchase_extractor_spec.rb

```diff
@@ -28,7 +28,8 @@
     )
   end
 
-  context "extract!" do
+  context "extracting a check" do
```

Comment:
It might make sense to wrap these next contexts in a `describe "#extract"` block.